LoopKit/Prescription.swift

```diff
@@ -0,0 +1,12 @@
+//
+//  Prescription.swift
+//  LoopKit
+//
+//  Created by Rick Pasetto on 7/22/20.
+//  Copyright © 2020 LoopKit Authors. All rights reserved.
+//
+
+public protocol Prescription {
```

Comment:
Will a type conforming to Prescription (e.g. Tidepool's impl) do something other than hold these two properties? If not, consider making this a struct

Reply:
I don't know the answer to this, yet. I'm not sure anyone does...which is specifically why I made it a `protocol`.